client/src/components/scans/scanListItem.js

```diff
@@ -205,6 +205,20 @@ class ScanListItem extends React.Component {
   renderActions() {
     const { item, onSummaryDownload, onDetailedDownload, onPause, onCancel, onStart, onResume } = this.props;
 
+    let downloadActions = null;
+    if (_.get(item, 'most_recent.report_id')) {
```

Comment:
This check and the one from `line 109 scanJobsList.js` seem like they should be the same

Reply:
The problem is in the JSX if the value is not a boolean then the number would be displayed. 1 and true are treated differently.

Reply:
Can see the truthy check, and it being passed on. Unsure follow the rest, we may be talking from different points. Regardless, looks like the same `_.get` is called a few times, something to iterate on, along with with the HTML... looking good